daemon/daemon.go

```diff
@@ -471,7 +471,11 @@ func (daemon *Daemon) restore() error {
 	}
 	group.Wait()
 
-	daemon.netController, err = daemon.initNetworkController(activeSandboxes)
+	// Initialize the network controller and configure network settings.
+	//
+	// Note that we cannot initialize the network controller earlier, as it
+	// needs to know if there's active sandboxes (running containers).
+	err = daemon.initNetworkController(activeSandboxes)
```

Comment:
Small nit: this could now be `if err := ...; err != nil {`

Reply:
done 👍